content/get-started.md

```diff
@@ -1,7 +1,23 @@
 ---
 title: Get Started with Webpack
 ---
+#### Why `webpack`?
 
-* basic concepts, get webpack up and running, technical focus
-* target - the user should be able to complete this in 10 minutes
+`webpack` is a module bundler.
+A bundler is a tool that builds all your project files to a single asset file that can be loaded onto a browser.
+It requires that your application files be written as [modules](/concepts/modules).
```

Comment:
I would merge these sentences so you get longer paragraphs. Reads a little better.